lib/p2p/packets/HelloPacket.ts

```diff
@@ -0,0 +1,30 @@
+import Packet from './Packet';
+import PacketType from './PacketType';
+
+type HelloPacketBody = {
+  version: string;
+  nodeKey: string;
+  listenPort: number;
+  pairs: string[];
+};
+
+class HelloPacket extends Packet {
+  public type: PacketType = PacketType.HELLO;
+  public responseType = PacketType.HELLO;
+  public responseTimeout = 10000;
```

Comment:
All these properties seem like they should be static, since they shouldn't be subject to change and they are inherent to the class. Same goes for other classes that extend `Packet`. I think the best way to do this would be to make these getter functions that return a fixed packet type or number. The getter methods can be defined on the base `Packet` class and then overridden by the child classes.

Reply:
You're right. That's the bad class structure I was talking about. I originally did it this way to avoid static properties inheritance issues. I'll do some experiments with it, in addition to what you suggested, and fix this design in the next PR.